backend/dataall/modules/s3_datasets_shares/services/s3_share_managed_policy_service.py

```diff
@@ -128,6 +128,23 @@ def check_resource_in_policy_statement(target_resources: list, existing_policy_s
                 return False
         return True
 
```

Comment:
nit: function name to `check_s3_actions_in_policy_statement` as the other functions in the class